src/Content.js

```diff
@@ -1,10 +1,49 @@
 import React from "react";
+import styled, { ThemeProvider } from "styled-components";
 import Home from "./components/Home.js";
+import Products from "./components/Products";
 
-function Content() {
+// Define our `fg` and `bg` on the theme
+const theme = {
+  fg: "#00004f",
+  bg: "white",
+};
+
+// This theme swaps `fg` and `bg`
+const invertTheme = ({ fg, bg }) => ({
+  fg: bg,
+  bg: fg,
+});
+
+const Button = styled.button`
+  color: ${(props) => props.theme.fg};
+  border: 2px solid ${(props) => props.theme.fg};
+  background: ${(props) => props.theme.bg};
+
+  font-size: 1em;
+  margin: 1em;
+  padding: 0.25em 1em;
+  border-radius: 3px;
+`;
+
+const If = ({ conditional, children }) => {
+  if (conditional) return children;
+  return null;
+};
```

Comment:
Since you are not using this function you can delete it to clean your code

Reply:
Thanks, gonna remove it for now and use it later if needed.